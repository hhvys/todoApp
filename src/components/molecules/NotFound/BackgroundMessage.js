import React from 'react';
import styles from './BackgroundMessage.mod.scss';

const BackgroundMessage = ({message}) => (
	<div className={`${styles.notFound} d-flex align-items-center justify-content-center flex-column fixed`}>
		<h3 className={`${styles.h3} mt-3`}>{message}</h3>
	</div>
);


export default BackgroundMessage;
