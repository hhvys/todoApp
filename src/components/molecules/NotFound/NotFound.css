.div__not__found {
    background: rgba(0,0,0,0.4);
    height: 200px;
    width: 30%;
    border-radius: 10px;
    top: 30%;
    left: 40%;
}