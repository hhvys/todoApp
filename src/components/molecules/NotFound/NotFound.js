import React from 'react';
import styles from './NotFound.scss';

const NotFound = () => (
	<div className={`${styles.notFound} d-flex align-items-center justify-content-center flex-column fixed`}>
		<h3 className={`${styles.h3} mt-3`}>No Search Results</h3>
	</div>
);


export default NotFound;
