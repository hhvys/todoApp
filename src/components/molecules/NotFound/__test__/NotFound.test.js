import React from 'react';
import NotFound from '../NotFound';

describe('component', () => {
	describe('NotFound', () => {
		it('should render', () => {
			expect(toJSON(shallow(<NotFound/>))).toMatchSnapshot();
		});
	});
});
