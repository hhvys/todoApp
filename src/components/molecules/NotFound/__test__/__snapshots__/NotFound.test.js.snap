// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component NotFound should render 1`] = `
<div
  className="notFound d-flex align-items-center justify-content-center flex-column fixed"
>
  <h3
    className="h3 mt-3"
  >
    No Search Results
  </h3>
</div>
`;
